src/kvstore/Common.h

```diff
@@ -29,6 +29,11 @@ enum ResultCode {
     ERR_TAG_NOT_FOUND       = -11,
     ERR_EDGE_NOT_FOUND      = -12,
     ERR_ATOMIC_OP_FAILED    = -13,
+    // We list the failure about atomic operation below
```

Comment:
This is the reason why i give up this approach when reviewing your last PR.

Reply:
Emmm, really sad trade-off.